packages/lib/src/components/helpers/OpenInvoiceContainer/OpenInvoiceContainer.tsx

```diff
@@ -2,8 +2,19 @@ import { h } from 'preact';
 import UIElement from '../../UIElement';
 import OpenInvoice from '../../internal/OpenInvoice';
 import CoreProvider from '../../../core/Context/CoreProvider';
+import { UIElementProps } from '../../types';
+import { AddressSpecifications } from '../../internal/Address/types';
 
-export default class OpenInvoiceContainer extends UIElement {
+interface OpenInvoiceElementProps extends UIElementProps {
+    consentCheckboxLabel?: h.JSX.Element;
+    billingAddressRequiredFields?: string[];
+    billingAddressSpecification?: AddressSpecifications;
+
+    // TODO: add other props for OpenInvoiceElement
```

Comment:
I have made a similar thing for RatePay, this looks more complete though. We should check when merging both to master.